Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -140,15 +140,15 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
 ```bash
 exec "$SHELL"
 ```
-4. Now you can install Python 3.8.15 and the associated pip like this:
+4. Now you can install Python 3.9 and the associated pip like this:
```

Comment:
Hi @ProfessionalCaddie, thanks for your PR!

Just a note, for anywhere there was a patch version originally (3.8.15), please replace with 3.9,20 instead of just 3.9. This is for consistency, since we need to make sure that everyone is using the same specific version of Python that is used in the production server. Thanks!